### PR TITLE
Add a table with font-weight and font-style alongside url

### PR DESCRIPTION
**Pull request by jamie-lynch**
## What does this change?

This PR updates the README to now include the font-weight and font-style values alongside each font URL

### CODE REVIEW

**Comment by sndrs** [comment]
i would be good to make it clear the weight and style are the recommended values for CSS (they're not intrinsic to the files and depend on the `@font-face` declarations on a page).

File: README.md [hunk 1]
```diff
@@ -17,44 +18,53 @@ Also consider using https://github.com/guardian/auto-foft.
 
 All of the files in [`fonts/web`](fonts/web) are available from `https://assets.guim.co.uk/static/frontend/fonts/`.
 
-These are the URLs that should be used on all Guardian websites. All URLs will also work for `.woff` and `.ttf`.
+These are the URLs that should be used on all Guardian websites. All URLs will also work for `.woff` and `.ttf`. The recommended CSS values for `font-weight` and `font-style` are shown for each URL.
```
Comment (jamie-lynch) [inline README.md:21]: @sndrs Is this line enough?

Comment (sndrs) [reply README.md:21]: yeah i reckon 👍

**Review by sndrs — APPROVED** [review]
great!

**Pull request merged**